### ISSUE TITLE
Is Nethunter still happening?

### ISSUE THREAD

**Issue by ozsteyr** [offensive-security/kali-nethunter#1157]
I'm asking because 6 months without a commit and a long time with unresolved issues.

Is NH still relevant when comparing to the many pi's and God type products?

I personally prefer NH to all others. 

What do you Devs think?

**Comment by 0E800** [comment]
Pretty sure anyone serious enough about information security should know quite a bit about Linux and has a $ making workflow with tried and proven methods that yield results regardless of the hardware/device/OS the tools are being executed from.

There is enough information available between the world wide web and this github issue thread for anyone with the ability to use a mouse and keyboard. However, most importantly; reading comprehension and a unyielding determination to stick with it to see it through till you grok it is key.

Tldr,  any android device is capable to run programs required. What does it matter the year your hammer came out or what its made of, so long as you can hit a nail with it...

**Comment by 0E800** [comment]
Yes nethunter is still relevant.

Which was what your question was.

The reason the answer is yes, is the same reason why a car without air in the tire is still a vehicle even if owner cannot make it work.
